### PR TITLE
cf-component-text to fela migration

### PR DESCRIPTION
**Pull request by sejoker**

### CODE REVIEW

File: packages/cf-component-text/src/TextTheme.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  'textTransform--capitalize': 'capitalize',
+  'textTransform--capitalize:first-word': 'capitalize',
+  'textTransform--lowercase': 'lowercase',
+  'textTransform--uppercase': 'uppercase',
+  'color--info': '#00a9eb',
+  'color--success': '#68970f',
+  'color--warning': '#fca520',
+  'color--error': '#ff3100',
+  'color--muted': '#gray'
+});
```
Comment (jwineman) [inline packages/cf-component-text/src/TextTheme.js]: We can move the colors here: https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js

Comment (sejoker) [inline packages/cf-component-text/src/TextTheme.js]: TextTheme styles are specific to our examples styles. Do we want them to be part of the cf-style-const?

Comment (tajo) [inline packages/cf-component-text/src/TextTheme.js]: Yes please. Also, I would rename them to `colorWarning`, `colorError` ... etc. So it's JS compatible.

Comment (sejoker) [inline packages/cf-component-text/src/TextTheme.js]: I can rename to `colorwarning`, `colorerror` easily, but camelCase format will require more work, is it worth the effort?

Comment (tajo) [inline packages/cf-component-text/src/TextTheme.js]: Yea, we should stay consistent and use camelCase everywhere. It's easy to do it now than later.

File: packages/cf-component-text/src/TextTheme.js [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+    lineHeightSmall: 1.3,
+    textAlignStart: 'left',
+    textAlignCenter: 'center',
+    textAlignJustify: 'justify',
+    textAlignEnd: 'right',
+    textTransformCapitalize: 'capitalize',
+    'textTransformCapitalize:first-word': 'capitalize',
+    textTransformLowercase: 'lowercase',
+    textTransformUppercase: 'uppercase',
+    colorInfo: '#00a9eb',
```
Comment (jwineman) [inline packages/cf-component-text/src/TextTheme.js]: `const`?

Comment (sejoker) [inline packages/cf-component-text/src/TextTheme.js]: @jwineman my bad, updated PR

**Pull request merged**